tests/e2e/kind/multi-cp/multi-cp.yaml

```diff
@@ -0,0 +1,9 @@
+# a single control-plane cluster with 3 workers
```

Comment:
@fabriziopandini can we substitute single control-plane with multiple control-plane?

Reply:
@fabriziopandini wdyt?